artichoke-backend/src/value.rs

```diff
@@ -249,6 +249,68 @@ impl ValueLike for Value {
         }
     }
 
+    fn unchecked_funcall(
+        &self,
+        func: &str,
+        args: &[Self::Arg],
+        block: Option<Self::Block>,
+    ) -> Result<Value, ArtichokeError> {
+        self.interp.0.borrow_mut();
+        // Ensure the borrow is out of scope by the time we eval code since
+        // Rust-backed files and types may need to mutably borrow the `Artichoke` to
+        // get access to the underlying `ArtichokeState`.
+        let (mrb, _ctx) = {
+            let borrow = self.interp.0.borrow();
+            (borrow.mrb, borrow.ctx)
+        };
+
+        let _arena = self.interp.create_arena_savepoint();
```

Comment:
Since we’re explicitly dropping the arena index, let’s remove the underscore from the name